packages/backend/src/accounting/tigerbeetle/accounts.ts

```diff
@@ -4,23 +4,30 @@ import {
   CreateAccountError as CreateAccountErrorCode
 } from 'tigerbeetle-node'
 
-import { ServiceDependencies, TigerbeetleAccountCode } from './service'
+import { ServiceDependencies, TigerBeetleAccountCode } from './service'
 import { TigerbeetleCreateAccountError } from './errors'
-import { AccountId, toTigerbeetleId } from './utils'
+import { AccountId, toTigerBeetleId, AccountUserData128 } from './utils'
 
 export interface CreateAccountOptions {
   id: AccountId
   ledger: number
-  code: TigerbeetleAccountCode
+  code: TigerBeetleAccountCode
+  // This flag links the result of this account creation to the result of the next one in the request,
+  // such that they will either succeed or fail together.
+  // The last account in a chain of linked accounts does not have this flag set.
+  linked?: boolean
+  // When set, the account will retain the history of balances at each transfer.
+  history?: boolean
+  userData128: AccountUserData128
```

Comment:
Make use of the 16-byte userData to store the bilateral relationship between debtor/creditor.